aiohttp/multidict.py

```diff
@@ -305,6 +305,9 @@ def __init__(self, items):
     def __len__(self):
         return len(self._items)
 
+    def __repr__(self):
+        return '{0.__class__.__name__}({0._items!r})'.format(self)
```

Comment:
I just copied the method from `abc`.
I believe its not the most common case to `print(request.GET.items())` so I wasn't thinking how it would be more informative for end-user. I guess it can be kept as is